pages/client-libraries/go.mdx

```diff
@@ -30,14 +30,39 @@ Necessary prerequisites that should be installed in your local environment are:
 
 ### Run Memgraph 
 
-Start by running Memgraph in the background. The easiest way to install and run Memgraph is by using Docker. Start the container with the following command:
-
-<CodeSnippet code="docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 --name memgraph memgraph/memgraph-platform" page="client-libraries/go"/>
-
-The command above will start [Memgraph Platform](/getting-started/install-memgraph/docker#install-memgraph-platform), which includes [Memgraph database](/getting-started/install-memgraph/docker#other-available-docker-images), [Memgraph Lab](/data-visualization#quick-start) and [Memgraph MAGE](/advanced-algorithms#quick-start). Memgraph uses Bolt protocol to communicate with the client using the exposed 7687 port. 
-Memgraph Lab is a web application you can use to visualize the data. It's accessible at [http://localhost:3000](http://localhost:3000) if Memgraph Platform is running correctly. The 7444 port enables Memgraph Lab to access and preview the logs, which is why both of these ports need to be exposed. 
-
-For more information visit the getting started guide on [how to run Memgraph with Docker](/getting-started/install-memgraph/docker).
+If you're new to Memgraph or you're in a developing stage, we
+recommend using the Memgraph Paltform. Besides the database, it also
```

Comment:
```suggestion
recommend using the Memgraph Platform. Besides the database, it also
```

Fix the same typo in all driver pages. Please use Grammarly so such errors don't occur this often or consider adding some automatic check on this repository.

Reply:
Fixed.